src/App.jsx

```diff
@@ -1,11 +1,32 @@
+//! Imports
+import { Spinner } from 'react-bootstrap'
```

Comment:
can you use, your own spinner, please. can be return '{ Spinner } from 'react-bootstrap''